.travis.yml

```diff
@@ -11,3 +11,7 @@ branches:
   only:
     - master
     - 1.x_dev
+
+script:
+  - xcodebuild -workspace push-sdk.xcworkspace -scheme push-sdk -sdk iphonesimulator build
+  - xcodebuild -workspace push-sdk.xcworkspace -scheme push-sdk -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.0' test
```

Comment:
just curious what makes these script lines necessary on iOS9 - related to the update or just general improvement as part of the update?